tests/unit/views/pacbio/PacbioRunIndex.spec.js

```diff
@@ -1,15 +1,41 @@
 import PacbioRunIndex from '@/views/pacbio/PacbioRunIndex'
 import Response from '@/api/Response'
 import { mount, store, Data, flushPromises, nextTick } from '@support/testHelper'
+import { createTestingPinia } from '@pinia/testing'
```

Comment:
Just trying to get my head round the differences between creating an active pinia and using the testing Pinia so not entirely sure what is going on here?

Reply:
`createPinia` is for creating a Pinia instance to test pinia stores and `createTestingPinia` is to mock Pinia store to test a component that uses Pinia store. 
This is how it is recommended in Pinia documentation https://pinia.vuejs.org/cookbook/testing.html